cmd/calendar_demo/main.go

```diff
@@ -20,9 +20,12 @@ func main() {
 	l.Alignment = fyne.TextAlignCenter
 	d := &date{instruction: i, dateChosen: l}
 
-	calendar := xWidget.NewCalendar(time.Now(), d.onSelected, 32)
+	c := container.NewVBox(i, l)
```

Comment:
Why bring this definition forward? It could be created at the end as it was before could it not? Fewer lines of code and avoid the `append`.

Reply:
Good point, I don't know I changed that.